### ISSUE TITLE
check_syntax: not structured data defined table

### ISSUE THREAD

**Issue by ed-holland** [abaplint/abaplint#3361]
Not sure what is wrong here. The Table BDCLM is part of the dependency repository.

```
node version:     v16.20.2
abaplint version: 2.111.0
```

Message
`./src/nw/#jcs#bc_rsbdcsub_status.prog.abap[37, 1]                            - not structured, bdclm (check_syntax) [E]`

Code
![image](https://github.com/user-attachments/assets/7daeab62-2827-41a4-b3c0-371065a7d386)

Playground
```
REPORT zfoobar.

DATA:
  BEGIN OF ls_apqle.
    INCLUDE STRUCTURE bdclm.
DATA:
    msgv1    TYPE symsgv,
    msgv2    TYPE symsgv,
    msgv3    TYPE symsgv,
    msgv4    TYPE symsgv,
    longtext TYPE bdc_mpar,
  END OF ls_apqle.

WRITE ls_apqle.
```

**Comment by larshp** [comment]
@ed-holland try latest version, 1.112.0, the syntax error reporting has been rewritten, should give less errors ie. the important ones first

**Comment by ed-holland** [comment]
@larshp tried the new version `abaplint version: 2.112.1`

There was no change that i noticed. This is our complete error list
```
./src/ecc/#jcs#fi.fugr.#jcs#fi_sl_user_table_fields.abap[26, 5]               - Read table, not a table type (check_syntax) [E]
./src/ecc/#jcs#fi.fugr.#jcs#fi_sl_user_table_fields.abap[38, 5]               - Read table, not a table type (check_syntax) [E]
./src/ecc/#jcs#hcm.fugr.#jcs#hr_read_infotype.abap[106, 26]                   - Definition for "CL_ABAP_STRUCTDESCR" not found in scope(AttributeName) (check_syntax) [E]
./src/fcc/#jcs#fcc.fugr.#jcs#fcc_task_documents.abap[1, 10]                   - Variable "NAME" contains unknown: /JCS/FCC_TSK_ITEM-TEXT not found, lookup (unknown_types) [E]
./src/nw_shared/#jcs#nw_shared_sync.fugr.#jcs#bc_rfc_read_table.abap[446, 50] - Definition for "CL_ABAP_TABLEDESCR" not found in scope(AttributeName) (check_syntax) [E]
./src/nw/#jcs#bc_rsbdcsub_status.prog.abap[39, 23]                            - not structured, bdclm (check_syntax) [E]
./src/nw/#jcs#bc_rtts.prog.abap[54, 37]                                       - Method "get_table_line_type" not found, methodCallChain (check_syntax) [E]
./src/nw/#jcs#bc_rtts.prog.abap[54, 37]                                       - Method "get_table_line_type" not found, methodCallChain (check_syntax) [E]
./src/nw/#jcs#bc_rtts.prog.abap[59, 42]                                       - Method "get_components" not found, methodCallChain (check_syntax) [E]
./src/nw/#jcs#bc_rtts.prog.abap[59, 42]                                       - Method "get_components" not found, methodCallChain (check_syntax) [E]
./src/nw/#jcs#bc.fugr.#jcs#bc_xbp_job_add_abap_step.abap[489, 32]             - "valid_pri_params" not found, Target (check_syntax) [E]
./src/nw/#jcs#sql.fugr.#jcs#lsqlf01.abap[72, 45]                              - Definition for "CL_ABAP_TABLEDESCR" not found in scope(AttributeName) (check_syntax) [E]
./src/nw/#jcs#sql.fugr.#jcs#lsqlf02.abap[86, 37]                              - Method "get_table_line_type" not found, methodCallChain (check_syntax) [E]
./src/nw/#jcs#sql.fugr.#jcs#lsqlf02.abap[91, 42]                              - Method "get_components" not found, methodCallChain (check_syntax) [E]
./src/nw/#jcs#sql.fugr.#jcs#sql_read_view.abap[154, 45]                       - Definition for "CL_ABAP_TABLEDESCR" not found in scope(AttributeName) (check_syntax) [E]
/tmp/abaplint-LjCnCo/src/sbdc/rsbdcil1.prog.abap[35, 28]                      - not structured, BDCLM (check_syntax) [E]
abaplint: 16 issue(s) found, 972 file(s) analyzed
```
